CHANGELOG.md

```diff
@@ -14,11 +14,7 @@ Main (unreleased)
 
 - Have `loki.echo` log the `entry_timestamp` and `structured_metadata` for any loki entries received (@dehaansa)
 
-- Update mysqld_exporter to v0.17.1, most notable changes: (@cristiangreco)
```

Comment:
Did this accidentally get dropped?

Reply:
Oops, must have had a bad changelog merge and didn't realize.